### ISSUE TITLE
js-hyperx and vomit library

### ISSUE THREAD

**Issue by bredele** [mappum/js-hyperx#1]
Hey @mappum I was wondering if js-hyperx would work with my library (http://github.com/bredele/vomit) or yoyo (http://github.com/naxogden/yo-yo). Could you please confirm?

**Comment by mappum** [comment]
Yes, it should work. But right now it is hardcoded to only do the HTML syntax highlighting if the template function name is `hx`. I'm still not sure the best way to support other function names, but I'm thinking about making it either autodetect HTML inside any template string and applying the highlighting to that, or just supporting more hardcoded function names (`vomit`, `html`, `yo`, etc.). Any thoughts about a good way to handle this?

**Comment by bredele** [comment]
@mappum I am not really familiar with atom configuration but you could use the the regexp `'begin': '[^A-Fa-f0-9_\\-^]\w`'` to target all words before the back-tick.

**Comment by mappum** [comment]
Right, but I'd prefer not to match all words because then it will attempt
to do HTML highlighting for any template string that uses a function, and
you might have some which are not HTML.
